docs/source/conf.py

```diff
@@ -101,7 +112,7 @@
 
 # General information about the project.
 project = "Finch"
-copyright = "2018-2020, David Huard"
+copyright = "2018-2022, David Huard"
```

Comment:
Let's get future-proof:
```suggestion
copyright = f"2018-{date.today().year()}, David Huard"
```